core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
```

Comment:
Another setting which can be useful is `ignore_review_state: true` ([ref](https://pkg.go.dev/k8s.io/test-infra/prow/plugins#Approve)). It disallows the GitHub review to impact the approvals of the PR.